maro/cli/inspector/cim_dashboard.py

```diff
@@ -0,0 +1,460 @@
+import math
+import os
+
+import altair as alt
+import numpy as np
+import pandas as pd
+import streamlit as st
+
+import maro.cli.inspector.common_helper as common_helper
+from maro.cli.inspector.common_params import CIMItemOption, ScenarioDetail
+from maro.cli.utils.params import GlobalFilePaths as Gfiles
+from maro.cli.utils.params import GlobalScenarios
+
+
+def generate_down_pooling_sample(down_pooling_num, start_epoch, end_epoch):
```

Comment:
since we have already specified the type of each argument in the docstring, why not add them in the declaration part for better reading?
e.g. def generate_down_pooling_sample(down_pooling_num: int, start_epoch: int, end_epoch: int) -> list: